packages/aila/src/utils/lessonPlan/fetchLessonPlanContentById.ts

```diff
@@ -18,8 +19,24 @@ export async function fetchLessonPlanContentById(
   if (!lessonPlanRecord) {
     return null;
   }
+
+  const upgradeResult = await upgradeQuizzes({
+    data: {
+      lessonPlan: lessonPlanRecord.content,
+    },
+    persistUpgrade: null,
+  });
+
+  // upgradeQuizzes expects and returns an object with a lessonPlan property
+  const upgradedLessonPlan =
+    upgradeResult.data &&
+    typeof upgradeResult.data === "object" &&
+    "lessonPlan" in upgradeResult.data
+      ? upgradeResult.data.lessonPlan
+      : null;
```

Comment:
Is there a need to error handle in case we end up with null here?

Reply:
Yeah you're right. Will fix